### DISCUSSION TITLE
XDM 7.2.8 Mac How to Run

### DISCUSSION THREAD

**Discussion by dmitry-pnrev** [subhra74/xdm#1145] [Q&A]
Hello. I have installed version 7.2.8 dmg format. But I don't understand where it is installed. I see that the script is installed, but how to use the application? I didn't find any information anywhere, even on the website. Sorry for bother

**Comment by Invictus2076** [comment]
Hi, I just had this problem while installing. Here's what you want to do. 

1. from desktop, look for Macintosh HD 
2. press CMD + Shift + "."
3. locate folder Volumes and open it
4. Open XDMSetup
5. Copy file xdm.app.tar.xz
6. paste it anywhere and open it 
7. It should extract into an app XDM. Paste the app into Applications folder and you're good to go
